### ISSUE TITLE
Update resources to the latest in Crowdin

### ISSUE THREAD

**Issue by hishitetsu** [files-community/Files#11241]
### What feature or improvement do you think would benefit Files?

Although the translation rate into Japanese is 100% in crowdin, many strings in Files are not localized.
I would like to see Crowdin's resources updated so that recently added strings can be translated.

### Requirements

Keep Crowdin's resources up to date.

### Files Version

2.4.33.0

### Windows Version

Windows 11 22H2 22621.1105

**Comment by Josh-65** [comment]
Crowdin translation have to be approved/review by someone official so that they are correct. Currently no one is assigned to it from what I know. Only after the approved can they be put into Files.

**Comment by Josh65-2201** [comment]
You can be the reviewer if you wanted. @yair100 Just has to send you a Crowdin invite I believe.

**Comment by hishitetsu** [comment]
My point was not that the latest translations are not reflected in the app, but that recently added strings are not in Crowdin and cannot be translated. Sorry for being so confusing.
I am not familiar with Crowdin, but who can add new strings to Crowdin to be translated?

**Comment by Josh-65** [comment]
Not sure, I would assume there taken from the resource files when updated in a PR and show when there merged into main.

**Comment by Josh-65** [comment]
Guess there could also be a bug with the syncing since there's also #11247

**Comment by yaira2** [comment]
Everything is in the en-us file, I'll verify the syncing is working.

**Comment by yaira2** [comment]
It seems that I have to update the file manually, not sure why this is the case but thank you for bringing it to my attention.

**Comment by hishitetsu** [comment]
Thank you @yair100 ! I have completed the translation of the new strings into Japanese.

**Comment by hishitetsu** [comment]
@yair100 How about using Github or Azure Pipeline integration?
https://store.crowdin.com/github
https://store.crowdin.com/azure-pipelines

**Comment by yaira2** [comment]
I fixed the setting, it was somehow turned off.

**Comment by hishitetsu** [comment]
I don't see the latest string in Crowdin, e.g. "Loading". Does it take time to reflect?

**Comment by hishitetsu** [comment]
Hmmm...no updates. The last update date seems Feb 13, 00:00 EST.
